activesupport/lib/active_support/lazy_load_hooks.rb

```diff
@@ -1,4 +1,7 @@
 module ActiveSupport
+  # These load hooks are how Rails communicates that a component of Rails like
```

Comment:
Is not this whole sentence already included in the next paragraph? `lazy_load_hooks allows Rails to lazily load`